feature-group-definitions/font-variant-alternates.yml

```
@@ -1,4 +1,5 @@
 name: font-variant-alternates
+description: The `font-variant-alternates` CSS property controls the usage of alternate glyphs for characters, as defined in `@font-feature-values` at-rules.
```

opened https://github.com/web-platform-dx/web-features/pull/847

```suggestion
description: The `font-variant-alternates` CSS property selects which alternate glyphs a font should use, as defined by `@font-feature-values` at-rules.
```